layers/+vim/evil-better-jumper/funcs.el

```diff
@@ -0,0 +1,38 @@
+;;; funcs.el --- evil-better-jumper layer functions file for spacemacs
+;;
+;; Copyright (c) 2012-2020 Sylvain Benner & Contributors
+;;
+;; Author: Thanh Vuong <thanh@gmail.com>
+;; URL: https://github.com/thanhvg
+;;
+;; This file is not part of GNU Emacs.
+;;
+;; Taking from Doom project core/core-editor.el
+;;; License: GPLv3
```

Comment:
GPLv3 must be replaced with the proper GPLv3 license header, see other layers for details.